scripts/send_digest.py

```diff
@@ -54,7 +54,7 @@ def send_digest(grouped_digests):
         digest_notification_ids = [message['_id'] for message in info]
         sorted_messages = group_messages_by_node(info)
 
-        if sorted_messages:
+        if sorted_messages['children']:
```

Comment:
Is there any possibility that `sorted_messages` might not have the `children` key?